packages/models-library/src/models_library/api_schemas_directorv2/dynamic_services_service.py

```diff
@@ -89,6 +90,11 @@ class RunningDynamicServiceDetails(ServiceDetails):
         alias="service_message",
     )
 
+    is_collaborative: Annotated[
```

Comment:
can you also remove the one present in the `DynamicSidecarServiceLabels` model since it will not be sued?